examples/dummy_plugin/dummy_plugin/__init__.py

```diff
@@ -2,14 +2,14 @@
 
 
 class DummyPluginConfig(PluginConfig):
-    name = "nautobot.extras.tests.dummy_plugin"
+    name = "dummy_plugin"
     verbose_name = "Dummy plugin"
     version = "0.0"
```

Comment:
I see that `pyproject.toml` declares the version as 1.0.0 - would be good to have them match, perhaps using:

```python

try:
    from importlib import metadata
except ImportError:
    # Python version < 3.8
    import importlib_metadata as metadata

__version__ = metadata.version(__name__)

...

    version = __version__
```

Reply:
Yes.